src/coreclr/jit/gtlist.h

```diff
@@ -147,9 +147,7 @@ GTNODE(COMMA            , GenTreeOp          ,0,GTK_BINOP|DBK_NOTLIR)
 GTNODE(QMARK            , GenTreeQmark       ,0,GTK_BINOP|GTK_EXOP|DBK_NOTLIR)
 GTNODE(COLON            , GenTreeColon       ,0,GTK_BINOP|DBK_NOTLIR)
 
-GTNODE(INDEX            , GenTreeIndex       ,0,GTK_BINOP|GTK_EXOP|DBK_NOTLIR) // SZ-array-element.
```

Comment:
`GT_INDEX` is also mentioned in `docs/design/coreclr/jit/{first-class-structs.md,ryujit-tutorial.md}`.

Reply:
Thank you for pointing that out!

I deleted some obsolete contents from `first-class-structs.md`, `ryujit-tutorial.md` required more extensive changes so I opened #69988 and will do it separately.